submissions/idowuMariam/html/product-page.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta charset="UTF8">
+    <title> Product page</title>
+<head>
+<body>
+    <nav>
+        <h1>Casual Outfits</h1>
+    </nav>
+    <section>
+        <h2>Tailored Jean Trousers</h2>
+        <img class="product 1" src="images\pic-1.jpg" width= "300" height="300">
+        <p class="product price">Price-$49.99</p>
+        <p class="product desription">Description: A casual outfit that can be worn at any time of the day, preferable for dinners and hangouts and also very comfy</p>
+    </section>
+    <section>
+        <h2> Sweatshirt</h2>
+        <img class="product 2" src="images\pic-2.jpg" width= "300" height="300">
+        <p class="product price">Price-$89.99</p>
+        <p class="product description"> Description: A casual outift that can be used to protect one's body against agressive sun rays and rain fall and can also be used for jogging during cold days</p>  
```

Comment:
This text is too long to be on one line. Kindly use your `enter` key on your keyboard to break it down into a new line